internal/audit/audit.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package audit
+
+import (
+	"crypto/md5"
+	"fmt"
+	"github.com/juju/loggo"
+	"strings"
+)
+
+var logger = loggo.GetLogger("audit")
+
+
+type AclOperation string
```

Comment:
Maybe this should be Operation as we might want to audit thingsa other that ACLs. If it isn't it should probably be spelled ACLOperation.